packages/react-core/src/components/Page/PageSection.tsx

```diff
@@ -28,6 +34,8 @@ export interface PageSectionProps extends React.HTMLProps<HTMLDivElement> {
   isFilled?: boolean;
   /** Limits the width of the section */
   isWidthLimited?: boolean;
+  /** Enable bottom/top sticky style to the page section */
+  sticky?: 'default' | 'bottom' | 'top';
```

Comment:
I think this is the same as @kmcfaul's PR? Or is this something different? https://github.com/patternfly/patternfly-react/pull/5025/files

Reply:
Oh yep, I can copy the relevant code or we can wait on @kmcfaul changes to be merged and I will do rebase.
Whatever suites you.

Reply:
Looks like #5025 was just merged, so you could use those props to fix these demos now! Thanks @boaz0!!

Reply:
Sweet thanks!!!